src/pages/what-is-ebpf.js

```diff
@@ -4,7 +4,7 @@ import React from "react";
 
 import "../stylesheets/index.scss";
 
-const pageMetaTitle = 'What is eBPF? An Introduction and Deep Dive into the eBPF Technology'
+const pageMetaTitle = 'What is extended Berkeley Packet Filter (eBPF)? An Introduction and Deep Dive into the eBPF Technology'
```

Comment:
I'm not against explaining the original meaning of the term eBPF, but I'm not sure this is the best place for it.

eBPF comes from extended BPF, which itself stands for Berkeley Packet Filter (or BSD Packet Filter depending on where you look). However, that acronym doesn't make much sense anymore for eBPF as it's not a packet filter anymore. So although the original acronym description is worth having, I don't think it should be in a title. It's no more used in the sense of its acronym than ZIP code or CARE package.

Reply:
Thanks for responding to my PR!  I appreciate your additional context, I'm new to both PRs and to eBPF.  I see now how it can be difficult to fit that information in, and I agree that the title isn't the best place for it.  I had recently read through https://developers.google.com/tech-writing/one/words#use_acronyms_properly which is where my reasoning for explaining the acronym comes from. 
Would the Introduction to eBPF h2 make for a better fit?